asab/library/providers/zookeeper.py

```diff
@@ -0,0 +1,169 @@
+import logging
+import asyncio
+
+import aiozk
+import aiozk.transaction
+
+from aiozk.recipes.base_watcher import BaseWatcher
+
+from .abc import LibraryProviderABC
+
+#
+
+L = logging.getLogger(__name__)
+
+#
+
+
+class ZooKeeperLibraryProvider(LibraryProviderABC):
```

Comment:
@bochkver Make sure you are synchronized with Mithun here. The provider will be used by all services and based on Kazoo.